scripts/cs2/run_compareScenarios2.R

```diff
@@ -56,9 +50,7 @@ run_compareScenarios2 <- function(
     cfgDefault = defaultConfigPath,
     outputDir = outfilepath,
     outputFile = outFileName,
-    outputFormat = "PDF",
-    reg = regionList,
-    mainReg = mainRegName
+    outputFormat = "PDF"
```

Comment:
`outputFormat = "PDF"` is the default of `remind2::compareScenarios2()` and does not need to be set here